stdlib/set.gr

```diff
@@ -665,8 +659,7 @@ provide module Immutable {
   let rec removeMin = node => {
     match (node) {
       Tree({ left: Empty, right, _ }) => right,
-      Tree({ key, left, right, _ }) =>
-        balancedNode(key, removeMin(left), right),
+      Tree({ key, left, right, _ }) => balancedNode(key, removeMin(left), right),
```

Comment:
You're right, it's 81. This is actually going to be incredibly tough to fix because of both how `concat_map` works and how all of the printing functions are written. The comma is not in the group with this branch because we specify it as a part of `~sep`.